adapters/platform_linux.c

```diff
@@ -47,8 +47,10 @@ const IO_INTERFACE_DESCRIPTION* platform_get_default_tlsio(void)
 #endif
 }
 
-STRING_HANDLE platform_get_platform_info(void)
+STRING_HANDLE platform_get_platform_info(PLATFORM_INFO_OPTION options)
 {
+    (void)options;
```

Comment:
could you add a brief comment why we are passing options to each platform even though we are not setting any options here. It makes sense in the PR but in a year I can see someone look at this and being slightly confused. Just something along the lines of: // no valid options so ignoring options parameter

Reply:
Sure, will do.